evoting/service/service.go

```diff
@@ -192,22 +192,22 @@ func (s *Service) Login(req *evoting.Login) (*evoting.LoginReply, error) {
 }
 
 // LookupSciper calls https://people.epfl.ch/cgi-bin/people/vCard?id=sciper
-// to convert scipers to names
+// to convert Sciper numbers to names.
 func (s *Service) LookupSciper(req *evoting.LookupSciper) (*evoting.LookupSciperReply, error) {
 	if len(req.Sciper) != 6 {
-		return nil, errors.New("sciper should be 6 digits only")
+		return nil, errors.New("Sciper should be 6 digits only")
```

Comment:
Errors are supposed to be lower-case, so that you can join them with other errors.